api/Document.json

```diff
@@ -708,6 +698,16 @@
                 "version_added": null,
                 "notes": "Read-only behavior"
               }
+            ],
+            "webview_android": [
+              {
+                "version_added": null,
```

Comment:
Read-write behavior is `false` for all three flavors of Chrome and both of Opera.

Reply:
Thanks! I also took this opportunity to split read-only and read-write behavior into sub-features for improved readability.